analyzers/nodejs/testdata/transitive-deps/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "babel-polyfill": "6.26.0"
+    "a": "a"
```

Comment:
this should map to some version that's valid given your other files (e.g. `1.0.0` or `^1.0.0`)

Reply:
Yea it should, thanks for catching this.